src/sv-pipeline/scripts/downstream_analysis_and_filtering/optimize_minGQ_ROC_v2.R

```diff
@@ -27,11 +27,11 @@ readTrioDat <- function(dn_dat){
   return(dat.out)
 }
 #Get counts of all inherited & de novo variants given min GQ
-filterGQStats <- function(dat,trio,minGQ=0){
+filterGQStats <- function(dat,minGQ=0){
   #Get variant counts
-  max.inh <- nrow(dat[which(dat$fam==trio & dat$label=="inh"),])
-  ret.inh <- nrow(dat[which(dat$fam==trio & dat$pro_GQ>minGQ & dat$label=="inh"),])
-  ret.dn <- nrow(dat[which(dat$fam==trio & dat$label=="dn" &
+  max.inh <- nrow(dat[which(dat$label=="inh"),])
```

Comment:
```suggestion
  max.inh <- sum(dat$label == "inh")
  ret.inh <- sum(dat$pro_GQ>minGQ & dat$label=="inh")
  ret.dn  <- sum(dat$label=="dn" &
                             dat$pro_GQ>minGQ & dat$fa_GQ>minGQ & dat$mo_GQ>minGQ))
```

It would probably be even faster to sum the boolean values returned by these queries (in R `TRUE` = 1 and `FALSE` = 0 in `sum`) and not form new intermediate data tables. Also simpler to read.

Reply:
Thanks R is definitely not my strong suit 😅 I would incorporate your suggestion except we've lost the test run to TTL and xuefang is on vacation, so I don't have a way to test right now. The 100x speedup (might've been even higher) was probably more dramatic since it was on gnomAD, so subsetting to the trio first saves iterating on the 140K rows 2 extra times.